benchmarks-examples/src/main/java/io/scalecube/benchmarks/examples/RampUpSimpleTestRunner.java

```diff
@@ -17,13 +17,13 @@
    */
   public static void main(String[] args) {
     BenchmarksSettings settings = BenchmarksSettings.from(args)
-        .injectors(1000)
-        .messageRate(10_000)
-        .rampUpDuration(Duration.ofSeconds(15))
-        .executionTaskDuration(Duration.ofSeconds(30))
+        .injectors(5)
```

Comment:
@snripa are we good with 5 instead of 1K?

Reply:
@aharonha it was just for mnual sanity test, we can put whatever convenient is for us.